creator-node/src/services/stateMachineManager/stateMonitoring/findSyncRequests.jobProcessor.js

```diff
@@ -110,14 +110,18 @@ module.exports = function ({
     for (const [secondary, resultCountMap] of Object.entries(
       secondaryToResultCountMap
     )) {
-      for (const [labelValue, metricValue] of Object.entries(resultCountMap))
+      for (const [labelValue, metricValue] of Object.entries(resultCountMap)) {
+        logger.info(
```

Comment:
i guess i didn't mean literally log when each metric is recorded, just that the 2 events should generally be paired, as opposed to logging during the record metric step istelf

idk if we have a similar log elsewhere (likely don't), but prob a log after the `findSyncsForUser()` call returns (maybe line 78) smth like `[issueSyncRequests][findSyncsForUser] primary= secondary= etc` with whatever standardized formatting makes sense

Reply:
also i think all the nested logic btwn lines 82-114 can be simplified now right? since we aren't tracking with same granularity

Reply:
if we just log primary+secondary and not the metric then how would we debug for example: if we saw CN5 showing a bunch of syncs failing for X reason and we want to find which primary syncing to which secondary is causing it? wouldn't there be a bunch of logs of just primary+secondary for syncs that had different results, so how would we distinguish which primary+secondary is causing the specific metric?

Reply:
the log would contain primary, secondary, status right? so we could regex in logs for that?

Reply:
also generally metrics are meant to be aggregate, e.g. if there are 100 sync failures, we don't need to know which of those 100 maps to which log, we just need to know that there are 100, and need the ability to find logs for all 100, if that makes sense?

Reply:
yeah 1 log per primary-secondary-result combo seems great, as long as we don't log for literally every user checked and only log for users that have some reason to be synced

Reply:
ok perfect! the code you mentioned between lines 82-114 should ensure it's aggregated and not logged for every user. I also just pushed something to not log the default status because that happens when we didn't need a sync (e.g., when this node isn't the user's primary) and we can deduce that happening anyway if no other result is logged